graveyard.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2016-05-15",
+    "dateOpen": "2012-05-02",
+    "description": "Revolv was a monitoring and control system that allowed users to control their connected devices from a single hub.",
+    "link": "https://en.wikipedia.org/wiki/Chromecast#Chromecast_Audio",
```

Comment:
Instead of `https://revolv.com`, is there another source?

Reply:
There's no wiki on the company, but there's several news articles.  Would those suffice?

Reply:
Yeah! That would be good. I'm trying to avoid future link rot, when possible.